node-it/src/test/scala/com/wavesplatform/it/sync/grpc/UpdateAssetInfoTransactionGrpcSuite.scala

```diff
@@ -0,0 +1,133 @@
+package com.wavesplatform.it.sync.grpc
+
+import com.google.protobuf.ByteString
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.account.KeyPair
+import com.wavesplatform.api.http.ApiError.{InvalidName, StateCheckFailed, TooBigArrayAllocation}
+import com.wavesplatform.it.NodeConfigs.Miners
+import com.wavesplatform.it.api.SyncHttpApi.assertApiError
+import com.wavesplatform.it.api.{Transaction, TransactionInfo}
+import com.wavesplatform.it.api.SyncHttpApi._
+import com.wavesplatform.it.sync._
+import com.wavesplatform.common.utils.{Base58, EitherExt2}
+import com.wavesplatform.it.sync.transactions.UpdateAssetInfoTransactionSuite
+import com.wavesplatform.it.sync.transactions.UpdateAssetInfoTransactionSuite.configWithUpdateIntervalSetting
+import com.wavesplatform.protobuf.transaction.{PBTransactions, Recipient}
+import io.grpc.Status.Code
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+import scala.util.Random
+import scala.concurrent.duration._
+
+class UpdateAssetInfoTransactionGrpcSuite extends GrpcBaseTransactionSuite with TableDrivenPropertyChecks{
+  import UpdateAssetInfoTransactionGrpcSuite._
+  val updateInterval = 2
+  override protected def nodeConfigs: Seq[Config] = Seq(configWithUpdateIntervalSetting(updateInterval).withFallback(Miners.head))
+
+  val issuer = KeyPair("issuer".getBytes)
+  val nonIssuer = KeyPair("nonIssuer".getBytes)
```

Comment:
Why not to use the predefined first/second addresses?

Reply:
fixed here https://github.com/wavesplatform/Waves/pull/2871/commits/471faf43564f012105b516053ceafbef113bbdcf